pkg/apis/io/v1alpha1/jaeger_types.go

```diff
@@ -150,6 +151,14 @@ type JaegerDependenciesSpec struct {
 	ElasticsearchNodesWanOnly   bool   `json:"elasticsearchNodesWanOnly"`
 }
 
+// JaegerEsIndexCleanerSpec holds the options related to es-index-cleaner
+type JaegerEsIndexCleanerSpec struct {
+	Enabled      bool   `json:"enabled"`
+	NumberOfDays int    `json:"numberOfDays"`
```

Comment:
Used 7 days as default. TTL in c* is 2 day though...

Reply:
The script supports other configuration properties, see https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/es/esCleaner.py#L11

index prefix, es hostname, I have not exposed them directly but they are derived from storage properties map.